src/main/java/com/google/devtools/build/lib/analysis/config/InvalidConfigurationException.java

```diff
@@ -45,17 +45,17 @@ public InvalidConfigurationException(String message, Throwable cause) {
   }
 
   public InvalidConfigurationException(Throwable cause) {
-    super(cause.getMessage(), cause);
+    super(cause == null ? null : cause.getMessage(), cause);
     this.detailedExitCode = null;
   }
 
   public InvalidConfigurationException(Code code, Throwable cause) {
-    super(cause.getMessage(), cause);
+    super(cause == null ? null : cause.getMessage(), cause);
     this.detailedExitCode = createDetailedExitCode(cause.getMessage(), code);
```

Comment:
One more, I think. Not sure if this is the right way to guard it (or if we can pass null as the 1st arg of createDetailedExitCode).
```suggestion
    this.detailedExitCode = cause == null ? null : createDetailedExitCode(cause.getMessage(), code);
```